CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.21.6+1
```

Comment:
0.21.6 was not released yet, no need for new version

Reply:
done